constant/barf.go

```diff
@@ -33,6 +33,9 @@ const (
 
 	// EnvPath is the path to the environment variables file
 	EnvPath = ".env"
+
+	// UseHTTPS enables barf use https by default. (Default value is false)
+	UseHTTPS = false
```

Comment:
How about we just put the comment as
```go
const (
...

// UseHTTPS enables barf to use https by default.
UseHTTPS = false
```
Notice there's a `to` now added in the above comment.